.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dockerComposeFile": "../dev.yaml",
```

Comment:
Preferred name for Compose file is `compose.yaml`: https://docs.docker.com/compose/compose-application-model/#the-compose-file
Please also move it under `.devcontainer`.

Reply:
Done.    The `compose.yaml` name is the default (doesn't require `-f`) and when it was at root of project I wasn't sure the dev compose was the "default" as you might have a demo of the project as the default compose (that's what I do).  Moving to .devcontainer makes it a non-issue.